distribution/lib/Standard/Base/0.0.0-dev/src/Data/Decimal.enso

```diff
@@ -89,6 +89,9 @@ type Decimal
     ## PRIVATE
     Value (big_decimal : BigDecimal)
 
+    ## PRIVATE
+    From_Float (big_decimal : BigDecimal) (original_value : Float)
```

Comment:
@jdunkerley @radeusgd I wonder if `Imprecise_Conversion` might be a better constructor here; a Decimal explicitly constructed from a Float does not use this constructor.